geopy/geocoders/googleplaces.py

```diff
@@ -123,7 +113,9 @@ def geocode(
             Currently, you can use components to filter by country.
             For example: components=country:fr would restrict your results to places within France.
         """
-        autocomplete_predictions = self.autocomplete(query, timeout, offset, location, radius, language, types, components)
+        autocomplete_predictions = self.autocomplete(query, timeout, location, radius, language, types, components)
+        if autocomplete_predictions and exactly_one:
+            autocomplete_predictions = [autocomplete_predictions[0]]
```

Comment:
according to the documentation form geopy if `exactly_one` is true we should return a single object, not a list of length one. (this is how the googlev3 geocoder works).

it seems rather stupid to me, but that's how geopy works and unless we really want to fork it (which i don't) we should follow its api.